src/Subsystems/Chassis.h

```diff
@@ -20,17 +24,30 @@ class Chassis: public frc::Subsystem {
 
 public:
 
+
+
 	enum ShifterValue{kShifterHigh = DoubleSolenoid::kForward, kShifterLow = DoubleSolenoid::kReverse};
+	enum AutoShifterValue{kAutoShiftOn = true, kAutoShiftOff = false};
```

Comment:
My mistake. This is fine then. 

I searched for `AutoShifterValue`, which is not used. You are correct that the enums are used.

No change required here.